src/algorithms/DominguezRios.jl

```diff
@@ -182,7 +182,7 @@ function optimize_multiobjective!(algorithm::DominguezRios, model::Optimizer)
             return status, nothing
         end
         _, Y = _compute_point(model, variables, f_i)
-        yN[i] = Y
+        yN[i] = Y + 1
     end
     MOI.set(model.inner, MOI.ObjectiveSense(), sense)
     ϵ = 1 / (2 * n * (maximum(yN - yI) - 1))
```

Comment:
Is another option something like this:
```suggestion
    ϵ = 1 / (2 * n * max(maximum(yN - yI) - 1, 1.0))
```

The issue is that if `maximum(yN - yI) <= 1` then ϵ is negative (or undefined)?

Reply:
Yes but even if this solves the issue with DominguezRios, KirlikSayin and TambyVanderpooten cannot find two nondominated points on the failing test case. They only find one point. Further overestimating nadir point is fine in my opinion and this is done in other algorithms like Benson & Sun where they compute y^hat > y_nadir.

Reply:
Okay. I don't really understand, but I'll defer to you on the implementation details :smile: